forest/cli/src/cli/snapshot_cmd.rs

```diff
@@ -30,6 +30,9 @@ use crate::cli::{cli_error_and_die, handle_rpc_err};
 pub(crate) const OUTPUT_PATH_DEFAULT_FORMAT: &str =
     "forest_snapshot_{chain}_{year}-{month}-{day}_height_{height}.car";
 
+pub(crate) const OUTPUT_PATH_DEFAULT_COMPRESSED_FORMAT: &str =
+    "forest_snapshot_{chain}_{year}-{month}-{day}_height_{height}.car.zst";
+
 #[derive(Debug, Subcommand)]
 pub enum SnapshotCommands {
     /// Export a snapshot of the chain to `<output_path>`
```

Comment:
Maybe change the documentation from `forest_snapshot_{chain}_{year}-{month}-{day}_height_{height}.car` to `forest_snapshot_{chain}_{year}-{month}-{day}_height_{height}.car(.zst)`

Reply:
Fixed